src/constants/routes.js

```diff
@@ -1,4 +1,6 @@
 export const LANDING = '/';
+export const MAIN = '/main';
+export const PROJECT = '/project';
 export const ADMIN = '/admin';
```

Comment:
Looks like I didn't check the routes file on some previous PR. admin,user and test_login routes are not needed